scripts/globals/status.lua

```diff
@@ -2912,7 +2913,7 @@ xi.animation =
 {
     NONE                    = 0,
     ATTACK                  = 1,
-    -- Death 2              = 2,
+    DESPAWN                 = 2,
```

Comment:
I didn't mean to commit this but it doesn't hurt anything. Just a missing entry for some reason.

Reply:
I'm not sure the naming is correct: when I try this on things they do their death animation, and don't despawn/fade out at all

Reply:
it probably expects to be paired with something or happen in order at the client

Reply:
This is what it is set to on the C++ side. This brings them in line; however, I'm happy to just revert this line change.

Reply:
_shrug_

core one prolly shouldn't have been done without some notation AND adding the same here. It really angers me when ppl desync the damned enumeration

Reply:
> I'm not sure the naming is correct: when I try this on things they do their death animation, and don't despawn/fade out at all

can be named it PLAY_DEAD like what dogs and cats do since they dont despawn lol

Reply:
> > I'm not sure the naming is correct: when I try this on things they do their death animation, and don't despawn/fade out at all
> 
> can be named it PLAY_DEAD like what dogs and cats do since they dont despawn lol

![image](https://user-images.githubusercontent.com/6871475/198895841-96eafd0e-eb35-4ade-94c0-c29cbadb507c.png)